ai/Gemini_ai_call_server.py

```
@@ -0,0 +1,846 @@
+                    speech_config=types.SpeechConfig(
+                        language_code="ko-KR",
+                        voice_config=types.VoiceConfig(
+                            prebuilt_voice_config=types.PrebuiltVoiceConfig(
+                                voice_name=self.selected_voice
+                            )
+                        )
+                    ),
+                    system_instruction=types.Part(
+                        text=(
+                            "너는 오늘 하루 일기 작성을 돕는 대화 도우미야."
+                            "사용자에게 오늘 어떤 일이 있었는지, 기분은 어땠는지, 기억에 남는 일은 무엇이었는지 자연스럽고 친근하게 차근차근 하나씩 질문해줘."
```

Pylint (3.3.7)

[convention] Line too long (103/100)

(C0301)